contact.html

```diff
@@ -0,0 +1,131 @@
+                </header>
+            </div>
+
+
+            
+            <div class="container content">
+                    <div>
+                        <div class="col-sm-12 col-md-6">
+                            <div class="row">
+                                <div class="col-xs-6 col-sm-4"><img src="img/doctor1.jpg" alt="" class="responsive"></div>
+                                <div class="col-xs-6 col-sm-8">
+                                    <h3>Dr. Smith</h3>
+                                </div>
+                                <div class="col-xs-12 col-sm-8">lorem25</div>
+                            </div>
+                            <div class="row">
+                                <div class="col-xs-6 col-sm-4"><img src="img/doctor2.jpg" alt="" class="responsive"></div>
+                                <div class="col-xs-6 col-sm-8">
+                                    <h3>Dr. Smith</h3>
+                                </div>
+                                <div class="col-xs-12 col-sm-8">lorem25</div>
+                            </div>
+                            <div class="row">
+                                <table class="table table-striped">
+                                    <thread>
+                                        <tr>
+                                            <th>Department</th>
+                                            <th>Phone</th>
+                                            <th>Extension</th>
+                                        </tr>
+                                    </thread>
+                                    <tbody>
+                                        <tr>
+                                            <td></td>
+                                            <td>834-987-0989</td>
+                                            <td>x890</td>
+                                        </tr>
+                                        <tr>
+                                            <td></td>
+                                            <td>834-987-0989</td>
+                                            <td>x890</td>
+                                        </tr>
+                                        <tr>
+                                            <td></td>
+                                            <td>834-987-0989</td>
+                                            <td>x890</td>
+                                        </tr>
+                                        <tr>
+                                            <td></td>
+                                            <td>834-987-0989</td>
+                                            <td>x890</td>
+                                        </tr>
+                                        <tr>
+                                            <td></td>
+                                            <td>834-987-0989</td>
+                                            <td>x890</td>
+                                        </tr>
```

Comment:
When you have the time, you need to add the actual info on the TD. In regular production, this would be a review that has a required action of now, but due to time we will go ahead and have you do it at a later date.